packages/mui/package.json

```diff
@@ -61,8 +63,7 @@
     "react-lorem-component": "^0.13.0",
     "react-scripts": "5.0.0",
     "resize-observer-polyfill": "^1.5.1",
-    "stylis-plugin-rtl": "^2.1.1",
-    "tslib": "^2.3.1"
+    "stylis-plugin-rtl": "^2.1.1"
```

Comment:
I think this package is required, so install either as dependency of peerDependency

Reply:
it only used in stories , package is allowed to use with or without using rtl in theme